dir.targets

```diff
@@ -45,4 +45,15 @@
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
   </PropertyGroup>
+
+  <!--
+    add the test-runtime project.json to the list of supplemental test data so it will get included
+    in the test archive; the stress test build system depends on it when building test mixes.
+  -->
+  <ItemGroup Condition="'$(IsTestProject)'=='true'">
```

Comment:
Good question. I cannot answer that without knowing why this is actually needed. I assume it is needed for stress runs for some reason or another and if that reason is unique to corefx then I'm fine with this if it isn't then we should share it.